internal/internal/src/footer.tsx

```diff
@@ -32,6 +32,10 @@ const footerProducts = [
     name: "NameKit",
     href: "https://namekit.io",
   },
+  {
+    name: "ENS Rainbow",
```

Comment:
```suggestion
    name: "ENSRainbow",
```

There's no space in "ENSRainbow".

Additionally, "ENSRainbow" needs to move up so it comes directly after "ENSNode".